openstack/block/api.go

```diff
@@ -559,6 +559,9 @@ func showVolumeDetails(bc *Context, w http.ResponseWriter, r *http.Request) (API
 	if err != nil {
 		return errorResponse(err), err
 	}
+	vol.MetaData = map[string]string{"contents": "not junk"}
+	s := "None"
+	vol.VolumeType = &s
```

Comment:
This was copied from an openstack example but it turns out it's not needed.  An empty map will suffice. I'll remove it.